scripts/lib/utils.ts

```diff
@@ -0,0 +1,163 @@
+import { strict as assert } from "assert";
+import { readFileSync, existsSync } from "fs";
+import { resolve, relative } from "path";
+
+import chalk from "chalk";
+import { addMinutes, formatISO } from "date-fns";
+import faker from "faker";
+import JSON5 from "json5";
+
+import { log } from "./log";
+import { SimConfig } from "./types";
+
+const INDEX_PADDING = 4;
+const INDEX_BASE = 10 ** INDEX_PADDING + 1;
```

Comment:
Care should be taken not to change these values while there are bot users in the DB or else they will not be targeted by the script (for use and/or removal afterwards).